.flowconfig

```diff
@@ -15,4 +15,5 @@
 [options]
 esproposal.class_static_fields=enable
 esproposal.class_instance_fields=enable
+esproposal.optional_chaining=enable
```

Comment:
redux-query has started using optional chaining operator ( `?` ) due to which we were getting the error while running flow. An [example](https://flow.org/en/docs/config/options/#toc-esproposal-optional-chaining). Here's the [reference](https://flow.org/en/docs/config/options/#toc-esproposal-optional-chaining) to solve this issue